backend/src/main/java/org/cryptomator/hub/api/DeviceResource.java

```diff
@@ -104,6 +104,7 @@ public Response remove(@PathParam("deviceId") @ValidId String deviceId) {
 
 	public record DeviceDto(@JsonProperty("id") @ValidId String id,
 							@JsonProperty("name") @NoHtmlOrScriptChars @NotBlank String name,
+							@JsonProperty("type") Device.Type type,
```

Comment:
Is there maybe an annotation for a default value, so we can get rid of the ternary operator?

Reply:
Haven't found one. Tried `@JsonProperty(value = "type", defaultValue = "DESKTOP")` and also `@JsonEnumDefaultValue` on the `DESKTOP` enum value but they didn't work.